core/operation/src/main/java/uk/gov/gchq/gaffer/operation/impl/get/GetElements.java

```diff
@@ -36,7 +36,6 @@
  * Gets elements from Gaffer based on {@link ElementId}s as
  * seeds and returns {@link uk.gov.gchq.gaffer.data.element.Element}s
  * There are various flags to filter out the elements returned.
- * See extensions of {@link GetElements} for further details.
  */
 public class GetElements implements
```

Comment:
Please could you update the class javadoc too, to include some information on how the different flags work together.